prow/config/jobs/envoy-1.9.yaml

```diff
@@ -54,6 +54,23 @@ jobs:
   image: gcr.io/istio-testing/build-tools:release-1.9-2021-02-04T20-04-44
   timeout: 4h
 
+- name: update-envoy
+  types: [periodic]
+  interval: 24h
+  command:
+  - ../test-infra/tools/automator/automator.sh
+  - --org=istio
+  - --repo=envoy
+  - "--title=Automator: merge upstream changes to $AUTOMATOR_ORG/$AUTOMATOR_REPO@$AUTOMATOR_BRANCH"
+  - --labels=auto-merge
+  - --modifier=merge_upstream_changes
+  - --token-path=/etc/github-token/oauth
+  - --cmd=VERSION="v$(cut --fields=1,2 -d. VERSION)"; git remote add upstream https://github.com/envoyproxy/envoy.git && git fetch upstream release/"${VERSION}" && git merge upstream/release/"${VERSION}"
```

Comment:
I have no idea what will happen afterwards. Is this merge commit going to be pushed as-is and used in a PR, or will the Automator try to create another commit or bail? I'm looking at [`automator.sh`](https://github.com/istio/test-infra/blob/master/tools/automator/automator.sh), and I don't see any support for this kind of workflow.

Reply:
This doesn't actually create a commit, just stages the changes for one; I'm not sure what will happen if there were conflicts during a merge; I'll take a closer look at automator.sh. Any chance we could test this on a dummy branch?